src/functions-reference/unbounded_continuous_distributions.Rmd

```diff
@@ -731,15 +806,16 @@ Generate a gumbel variate with location mu and scale beta; may only be
 used in transformed data and generated quantities blocks. For a description
 of argument and return types, see section
 [vectorized PRNG functions](#prng-vectorization).
+`r since("2.18")`
 
 ## Skew double exponential distribution
 
 ### Probability density function
 
 If $\mu \in \mathbb{R}$, $\sigma \in \mathbb{R}^+$ and $\tau \in [0, 1]$,
-then for $y \in \mathbb{R}$, 
-$$\begin{aligned} 
-& \text{SkewDoubleExponential}  (y|\mu,\sigma, \tau) = \\ 
+then for $y \in \mathbb{R}$,
+$$\begin{aligned}
+& \text{SkewDoubleExponential}  (y|\mu,\sigma, \tau) = \\
```

Comment:
[optional]
I strongly prefer`\mid` over `|` as it puts a bit of space around the vertical bar.

Also, it's conventional to have the start/end math symbols (here `$$`) be on their own line.

It's also sub-optimal to have lines wrap---better to break up expressions into natural chunks.

The use of `I` should be `\textrm{I}` because that's how special functions get set in math.

I prefer to use `(`  everywhere instead of using `{` or `[` in some places.  If they get confusing to parse, either think about rewriting in smaller chunks or using `\left(` and `\right)`.

Reply:
This also only shows up in the diff due to whitespace removal. I agree with your suggestions, but they seem best in a separate PR